mage/template.go

```diff
@@ -46,7 +46,7 @@ func main() {
 	defer func() {
 		err := recover()
 		if err != nil {
-			fmt.Println(err)
+			fmt.Printf("Error: %s\n", err)
```

Comment:
this and the one below need to be %v in case someone panics with something that isn't compatible with %s.